src/test/addrman_tests.cpp

```diff
@@ -296,23 +296,23 @@ BOOST_AUTO_TEST_CASE(addrman_find)
     addrman.Add(addr2, source2);
     addrman.Add(addr3, source1);
 
-    // Test 17: ensure Find returns an IP matching what we searched on.
+    // Test: ensure Find returns an IP matching what we searched on.
     CAddrInfo* info1 = addrman.Find(addr1);
     BOOST_CHECK(info1);
     if (info1)
```

Comment:
According to http://www.boost.org/doc/libs/1_64_0/libs/test/doc/html/boost_test/utf_reference/testing_tool_ref/assertion_boost_level.html you could use `BOOST_REQUIRE()`